Makefile

```diff
@@ -8,20 +10,28 @@ SPEC_PATH    := spec/
 
 .PHONY: default
 default: ci
-.PHONY: bash
-bash:
-	@$(COMPOSE_DEV) $(BASH)
 
 .PHONY: ci
 ci:
 	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
 
-.PHONY: clean
-clean:
-	rm -r data || true
-	$(COMPOSE_TEST) run vets-api rm -r coverage log tmp .git || true
+.PHONEY: ci-down
+ci-down:
 	$(COMPOSE_TEST) down
 
+.PHONY: ci-clean
+ci-clean:
```

Comment:
I don't see this target called anywhere, what gives?

Reply:
this was the `clean` target, but it wasn't needed anymore after I fixed the file permissions, but I left it in case it was in people's workflows.  I can remove it, or perhaps return an error?

Reply:
I've vote to remove. My understanding is that `COMPOSE_DEV` & `BASH_DEV` are for local use. Given that this target uses `COMPOSE_TEST`, it shouldn't be used locally by anyone.

Reply:
I removed this target